lib/resources/r4/observation.yaml

```diff
@@ -265,6 +461,77 @@ fields:
     </ul>
   url: https://hl7.org/fhir/R4/extension-event-performerfunction-definitions.html
 
+- name: performer.extension
+  required: 'No'
+  type: performerFunction
+  description: Distinguishes the type of involvement of the performer in the event.
+  action:
+    - terminology
+    - update
+  binding:
+    description: Codes identifying the function of the performer.
+    terminology:
+    - display: Performer function codes
+      system: http://terminology.hl7.org/CodeSystem/v3-ParticipationType
+      info_link: http://hl7.org/fhir/R4/valueset-performer-function.html
+    - display: Participation codes
+      system: http://terminology.hl7.org/CodeSystem/v2-0912
+      info_link: http://terminology.hl7.org/CodeSystem/v2-0912
+  example: |
+   {
+      "extension": [
+        {
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": 'http://terminology.hl7.org/CodeSystem/v3-ParticipationType',
+                "code": 'LA',
+                "display": 'legal authenticator'
+              }
+            ],
+            "text": 'legal authenticator'
+          },
+          "url": 'http://hl7.org/fhir/StructureDefinition/event-performerFunction'
+        }
+      ],
+    }
+  example2: |
+     {
+        "extension": [
+        {
+          "valueCodeableConcept": {
```

Comment:
formatting is off

Reply:
update: https://github.com/cerner/fhir.cerner.com/commit/be02d7b53316e773b9fbd31f46412de7954ed7a3#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcL502-R511

Reply:
it still looks off?

https://github.com/cerner/fhir.cerner.com/blob/R4-Observation-Labs-Update/lib/resources/r4/observation.yaml#L420
https://github.com/cerner/fhir.cerner.com/blob/R4-Observation-Labs-Update/lib/resources/r4/observation.yaml#L433

Reply:
adjusted the indentation : https://github.com/cerner/fhir.cerner.com/commit/b76dfa92bb8c78c2f3989f8e2fb777225d94d3b8#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR492-R514